src/StokesDrift.jl

```diff
@@ -39,44 +39,48 @@ abstract type AbstractStokesDrift end
 #####
 
 """
-    UniformStokesDrift{UZ, VZ, UT, VT} <: AbstractStokesDrift
+    UniformStokesDrift{P, UZ, VZ, UT, VT} <: AbstractStokesDrift
```

Comment:
```suggestion
    UniformStokesDrift{UZ, VZ, UT, VT, P} <: AbstractStokesDrift
```

Reply:
It's good for `P` to be first so we can dispatch on

```julia
const UnparameterizedUSD = UniformStokesDrift{<:Nothing}
```

Reply:
> It's good for `P` to be first so we can dispatch on
> 
> ```julia
> const UnparameterizedUSD = UniformStokesDrift{<:Nothing}
> ```

@glwagner I'm confused about this comment. I had put `P` first originally for this reason exactly (easier to dispatch) but changed because of your comment [here](https://github.com/CliMA/Oceananigans.jl/pull/3096#discussion_r1185557203). Are you suggesting I change it back?

Reply:
The order of type parameters and properties are independent. My comment was about properties, not type parameters.

Reply:
Ah, I see. Thanks for clarifying. It's done.